cmd/access/node_builder/access_node_builder.go

```diff
@@ -1401,10 +1404,16 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 			"program-cache-size",
 			defaultConfig.programCacheSize,
 			"[experimental] number of blocks to cache for cadence programs. use 0 to disable cache. default: 0. Note: this is an experimental feature and may cause nodes to become unstable under certain workloads. Use with caution.")
+
+		// Payer Balance
 		flags.BoolVar(&builder.checkPayerBalance,
 			"check-payer-balance",
 			defaultConfig.checkPayerBalance,
 			"checks that a transaction payer has sufficient balance to pay fees before submitting it to collection nodes")
+		flags.StringVar(&builder.checkPayerBalanceMode,
+			"check-payer-balance-mode",
```

Comment:
let's consolidate `check-payer-balance-mode` and `check-payer-balance`. You can remove `check-payer-balance`, and add an option `disabled` to `check-payer-balance-mode` which is the default.